README.md

```diff
@@ -222,6 +222,18 @@ When working with large-scale input data, it is important to create large enough
 TODO: Add an example for running on Google Cloud.
 
 ## Getting involved
+
+Currently, you need to install Python 3.6 and `svn` for developing Flax. After installing these prerequisites, you can clone the repository, set up your local environement, and run all tests with the following commands:
```

Comment:
Sorry for the silly question, but why do we need svn?

Reply:
For downloading tfds metadata (a directory inside the tensorflow/datasets repo).

Using git for downloading the directory would require us to clone the entire tensorflow/datasets repo (currently ~226M when cloning with `--depth=1`).

Reply:
Hmm interesting. In my VMs I never install svn and everything seems to work fine. Are you sure this is needed for all Flax development, or only if you want to use TFDS? (which is not a requirement for developing with Flax, I think)

Reply:
It's for running the examples.
If your VMs succeeded the `./tests/run_all_test.sh` then they either had a `.tfds` directory in the root or `svn` was installed on them by default.